app/src/main/java/org/simple/clinic/login/applock/AppLockScreen.kt

```diff
@@ -72,28 +61,21 @@ class AppLockScreen(context: Context, attrs: AttributeSet) : RelativeLayout(cont
 
     // The keyboard shows up on PIN field automatically when the app is
     // starting, but not when the user comes back from FacilityChangeScreen.
-    pinEditText.showKeyboard()
+    pinEntryCardView.pinEditText.showKeyboard()
```

Comment:
Why is it unnecessarily? I think it's okay to abstract properties when their underlying implementation can change. It looks fine here. 🤔

For instance, `forgotPinButton` is also exposed as a public field so that its visibility can be set to hidden by the screen. This will be used in `LoginPinScreen`.

Reply:
Hmm. Let's wait until this is used in multiple places before we decide.